decidim-proposals/spec/services/decidim/proposals/proposal_serializer_spec.rb

```diff
@@ -19,7 +19,8 @@ module Proposals
 
       let!(:proposals_component) { create(:component, manifest_name: "proposals", participatory_space: participatory_process) }
       let(:other_proposals) { create_list(:proposal, 2, component: proposals_component) }
-      let(:body) { Decidim::Faker::Localized.localized { ::Faker::Lorem.sentences(number: 3).join("\n") } }
+
+      let(:serialized) { described_class.new(proposal).serialize }
```

Comment:
```suggestion
      let(:serialized) { subject.serialize }
```

Reply:
As we're moving this from line 42, we should remove that line (as it is duplicated)